docs/changelog/changelog.mdx

```diff
@@ -3,6 +3,60 @@ title: Changelog
 description: All notable changes to Latitude.
 ---
 
+<Update label='2024-12-18'>
+## Latitude Telemetry
+
+We've released the 1.0 version of Latitude's Typescript SDK, which ships with
+support for [OpenLLMetry](https://github.com/traceloop/openllmetry). You can
+now trace all your LLM generations with a couple lines of code and start using
+Latitude's powerful prompt manager by creating prompts from your traces.
+
+We've made a quick video explaining the full process.
+
+(video)
+
+Here's the full list of supported providers/sdks at launch:
+
+- [Anthropic](/guides/sdk/tracing/anthropic)
+- [AWS Bedrock](/guides/sdk/tracing/aws-bedrock)
+- [Azure OpenAI](/guides/sdk/tracing/azure-openai)
+- [Cohere](/guides/sdk/tracing/cohere)
+- [Google AI Platform](/guides/sdk/tracing/google-ai-platform)
+- [OpenAI](/guides/sdk/tracing/openai)
+- [Vercel AI SDK](/guides/sdk/tracing/vercel-ai-sdk)
+- [Vertex AI](/guides/sdk/tracing/vertex-ai)
+
+## Version history
+
+We've added a new history section in projects. It displays the full history of prior committed versions of your project.
+
+![](/assets/changelog/241218/history-1.png)
+
+From a prompt, you can easily inspect its history by clicking on the history button at the top of the editor.
+
+![](/assets/changelog/241218/history-2.png)
+
+## Revert history
+
+Following up from the previous feature, you can now easily revert to a previous
+version of your project by clicking on the revert button in any prior version.
+
+![](/assets/changelog/241218/history-3.png)
+
+## File / Image prompt parameters
```

Comment:
Also promptl now has support for "file" content type, unlocking OpenAI input_audio audio files, Anthropic document PDF files and Google fileData audio, video and document files.